sdks/python/scripts/run_pytest.sh

```diff
@@ -29,10 +29,16 @@ posargs=$2
 
 # Run with pytest-xdist and without.
 pytest -o junit_suite_name=${envname} \
-  --junitxml=pytest_${envname}.xml -m 'not no_xdist' -n 6 --pyargs ${posargs}
+  --junitxml=pytest_${envname}.xml -m 'not no_xdist' -n 6 \
+  --ignore=apache_beam/runners/portability/flink_runner_test.py \
```

Comment:
Can you instead check if `test-pipeline-options` is set? I believe that's how all the other ITs are skipped (see `TestPipeline` code).

Reply:
Done